apple/LayoutReanimation/REASharedTransitionManager.m

```diff
@@ -341,7 +341,8 @@ - (void)swizzleScreensMethods
     Class screenClass = [RNSScreen class];
     Class screenViewClass = [RNSScreenView class];
     BOOL allSelectorsAreAvailable = [RNSScreen instancesRespondToSelector:viewDidLayoutSubviewsSelector] &&
-        [RNSScreenView instancesRespondToSelector:notifyWillDisappearSelector];
+        [RNSScreenView instancesRespondToSelector:notifyWillDisappearSelector] &&
+        [RNSScreenView instancesRespondToSelector:@selector(isModal)]; // used by REAScreenHelper
```

Comment:
Why do we need `@selector` here but we don't in the line above?

Reply:
We also use selectors but define a couple of lines above: https://github.com/software-mansion/react-native-reanimated/pull/5014/files#diff-b62ac3002c1da6d8f9df0d7d5cce57bd26cf59c1ceaeb77dcab4ac4475e2b093R340

`@selector(viewDidLayoutSubviews)` and `@selector(notifyWillDisappear)` are assigned to `SEL` object because I used them later, but `@selector(isModal)` is used only in one place.